### ISSUE TITLE
Server error [-3200] from 127.0.0.1:8000: IOU1: Startup-config startup-config could not be found on this server

### ISSUE THREAD

**Issue by lachlanjholmes** [GNS3/gns3-gui#41]
Hey.

Here are the logs and the pictures of the topology.

http://i.imgur.com/z04fQAE.png

Logs from gns3server running in terminal:
[lachlan@Ante-lappy ~]$ gns3server 
GNS3 server version 1.0a4
Copyright (c) 2007-2014 GNS3 Technologies Inc.
Running with Python 3.4.0 and has PID 2101
[I 140515 11:34:16 main:74] current locale is en_AU.UTF-8
[W 140515 11:34:16 config:82] no configuration file could be found or read
[I 140515 11:34:16 base:126] dynamips module running with PID 2103
[I 140515 11:34:16 base:79] ZeroMQ client (dynamips) connecting to 127.0.0.1:41214
[I 140515 11:34:16 base:126] vpcs module running with PID 2106
[I 140515 11:34:16 base:79] ZeroMQ client (vpcs) connecting to 127.0.0.1:41214
[I 140515 11:34:16 server:214] ZeroMQ server listening to 127.0.0.1:41214
[I 140515 11:34:16 base:126] iou module running with PID 2107
[I 140515 11:34:16 base:79] ZeroMQ client (iou) connecting to 127.0.0.1:41214
Starting server on 0.0.0.0:8000 (Tornado v3.2, PyZMQ v14.1.1, ZMQ v4.0.4)
[I 140515 11:35:01 jsonrpc_websocket:118] Websocket client a763f146-e2eb-4e1d-b283-872ec3823859 connected
[I 140515 11:35:01 web:1728] 200 GET /version (127.0.0.1) 0.92ms
[I 140515 11:35:07 **init**:228] saving iourc file content to /tmp/tmpe7zwwbhl
[I 140515 11:35:07 **init**:236] iouyap path set to /usr/local/bin/iouyap
[I 140515 11:35:07 **init**:240] this server is local with working directory path to /home/lachlan/GNS3/test/test-files
[I 140515 11:35:07 iou_device:282] IOU IOU1 [id=1]: working directory changed to /home/lachlan/GNS3/test/test-files/iou/device-1
[I 140515 11:35:07 iou_device:120] IOU device IOU1 [id=1] has been created
[I 140515 11:35:07 iou_device:282] IOU IOU2 [id=2]: working directory changed to /home/lachlan/GNS3/test/test-files/iou/device-2
[I 140515 11:35:07 iou_device:120] IOU device IOU2 [id=2] has been created
[I 140515 11:35:07 **init**:296] this server is local
[I 140515 11:35:07 **init**:263] starting the hypervisor manager with Dynamips working directory set to '/home/lachlan/GNS3/test/test-files/dynamips'
[I 140515 11:35:07 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7200 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:07 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7200.txt', '-H', '7200']
[I 140515 11:35:07 hypervisor:208] Dynamips process logging to /tmp/tmprzjjsctr
[I 140515 11:35:07 hypervisor:213] Dynamips started PID=2131
[I 140515 11:35:07 base:213] ZeroMQ client (iou) sending JSON-RPC custom error: Startup-config startup-config could not be found on this server for call id 5024dcf3-1062-45cd-afd9-52634d688622
[I 140515 11:35:07 base:213] ZeroMQ client (iou) sending JSON-RPC custom error: Startup-config startup-config could not be found on this server for call id 9d0a53eb-761b-441c-a8d8-ca56524963b6
[I 140515 11:35:07 hypervisor_manager:542] Dynamips server ready after 0.3580 seconds
[I 140515 11:35:07 hypervisor_manager:565] hypervisor 0.0.0.0:7200 has successfully started
[I 140515 11:35:07 router:112] router c2691 "R1" [id=1] has been created
[I 140515 11:35:07 router:453] router "R1" [id=1]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:07 router:688] router "R1" [id=1]: sparse memory enabled
[I 140515 11:35:07 router:655] router "R1" [id=1]: mmap enabled
[I 140515 11:35:07 router:1068] router "R1" [id=1]: console port updated from 2001 to 2101
[I 140515 11:35:07 router:1176] router "R1" [id=1]: MAC address updated from c001.0853.0000 to c001.0b22.0000
[I 140515 11:35:07 router:70] creating a new ghost IOS file
[I 140515 11:35:07 router:453] router "Ghost" [id=0]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:07 router:923] router "Ghost" [id=0]: ghost status set to 1
[I 140515 11:35:07 router:878] router "Ghost" [id=0]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:07 router:335] router "Ghost" [id=0] has been started
[I 140515 11:35:08 router:344] router "Ghost" [id=0] has been stopped
[I 140515 11:35:08 router:303] router "Ghost" [id=0] has been deleted (including associated files)
[I 140515 11:35:08 router:923] router "R1" [id=1]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R1" [id=1]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7201 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:08 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7201.txt', '-H', '7201']
[I 140515 11:35:08 hypervisor:208] Dynamips process logging to /tmp/tmpx0_ncryk
[I 140515 11:35:08 hypervisor:213] Dynamips started PID=2149
[I 140515 11:35:08 hypervisor_manager:542] Dynamips server ready after 0.0104 seconds
[I 140515 11:35:08 hypervisor_manager:565] hypervisor 0.0.0.0:7201 has successfully started
[I 140515 11:35:08 router:112] router c2691 "R2" [id=2] has been created
[I 140515 11:35:08 router:453] router "R2" [id=2]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:08 router:688] router "R2" [id=2]: sparse memory enabled
[I 140515 11:35:08 router:655] router "R2" [id=2]: mmap enabled
[I 140515 11:35:08 router:1068] router "R2" [id=2]: console port updated from 2001 to 2102
[I 140515 11:35:08 router:1176] router "R2" [id=2]: MAC address updated from c002.0865.0000 to c002.0b35.0000
[I 140515 11:35:08 router:923] router "R2" [id=2]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R2" [id=2]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7202 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:08 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7202.txt', '-H', '7202']
[I 140515 11:35:08 hypervisor:208] Dynamips process logging to /tmp/tmpj6pe0xo0
[I 140515 11:35:08 hypervisor:213] Dynamips started PID=2165
[I 140515 11:35:08 hypervisor_manager:542] Dynamips server ready after 0.0105 seconds
[I 140515 11:35:08 hypervisor_manager:565] hypervisor 0.0.0.0:7202 has successfully started
[I 140515 11:35:08 router:112] router c2691 "R3" [id=3] has been created
[I 140515 11:35:08 router:453] router "R3" [id=3]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:08 router:688] router "R3" [id=3]: sparse memory enabled
[I 140515 11:35:08 router:655] router "R3" [id=3]: mmap enabled
[I 140515 11:35:08 router:1068] router "R3" [id=3]: console port updated from 2001 to 2103
[I 140515 11:35:08 router:1176] router "R3" [id=3]: MAC address updated from c003.0875.0000 to c003.0b46.0000
[I 140515 11:35:08 router:923] router "R3" [id=3]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R3" [id=3]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7203 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:08 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7203.txt', '-H', '7203']
[I 140515 11:35:08 hypervisor:208] Dynamips process logging to /tmp/tmpvuds6qet
[I 140515 11:35:08 hypervisor:213] Dynamips started PID=2181
[I 140515 11:35:08 hypervisor_manager:542] Dynamips server ready after 0.0104 seconds
[I 140515 11:35:08 hypervisor_manager:565] hypervisor 0.0.0.0:7203 has successfully started
[I 140515 11:35:08 router:112] router c2691 "R4" [id=4] has been created
[I 140515 11:35:08 router:453] router "R4" [id=4]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:08 router:688] router "R4" [id=4]: sparse memory enabled
[I 140515 11:35:08 router:655] router "R4" [id=4]: mmap enabled
[I 140515 11:35:08 router:1068] router "R4" [id=4]: console port updated from 2001 to 2104
[I 140515 11:35:08 router:1176] router "R4" [id=4]: MAC address updated from c004.0885.0000 to c004.0b56.0000
[I 140515 11:35:08 router:923] router "R4" [id=4]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R4" [id=4]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7204 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:08 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7204.txt', '-H', '7204']
[I 140515 11:35:08 hypervisor:208] Dynamips process logging to /tmp/tmpf6eac9a_
[I 140515 11:35:08 hypervisor:213] Dynamips started PID=2197
[I 140515 11:35:08 hypervisor_manager:542] Dynamips server ready after 0.0107 seconds
[I 140515 11:35:08 hypervisor_manager:565] hypervisor 0.0.0.0:7204 has successfully started
[I 140515 11:35:08 router:112] router c2691 "R5" [id=5] has been created
[I 140515 11:35:08 router:453] router "R5" [id=5]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:08 router:688] router "R5" [id=5]: sparse memory enabled
[I 140515 11:35:08 router:655] router "R5" [id=5]: mmap enabled
[I 140515 11:35:08 router:1068] router "R5" [id=5]: console port updated from 2001 to 2105
[I 140515 11:35:08 router:1176] router "R5" [id=5]: MAC address updated from c005.0895.0000 to c005.0f69.0000
[I 140515 11:35:08 router:923] router "R5" [id=5]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R5" [id=5]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 hypervisor_manager:561] creating new hypervisor 0.0.0.0:7205 with working directory /home/lachlan/GNS3/test/test-files/dynamips
[I 140515 11:35:08 hypervisor:205] starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_log_7205.txt', '-H', '7205']
[I 140515 11:35:08 hypervisor:208] Dynamips process logging to /tmp/tmpp9dqc2pd
[I 140515 11:35:08 hypervisor:213] Dynamips started PID=2213
[I 140515 11:35:08 hypervisor_manager:542] Dynamips server ready after 0.0107 seconds
[I 140515 11:35:08 hypervisor_manager:565] hypervisor 0.0.0.0:7205 has successfully started
[I 140515 11:35:08 router:112] router c2691 "R6" [id=6] has been created
[I 140515 11:35:08 router:453] router "R6" [id=6]: has a new IOS image set: "/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image"
[I 140515 11:35:08 router:688] router "R6" [id=6]: sparse memory enabled
[I 140515 11:35:08 router:655] router "R6" [id=6]: mmap enabled
[I 140515 11:35:08 router:1068] router "R6" [id=6]: console port updated from 2001 to 2106
[I 140515 11:35:08 router:1176] router "R6" [id=6]: MAC address updated from c006.08a5.0000 to c006.0f79.0000
[I 140515 11:35:08 router:923] router "R6" [id=6]: ghost status set to 2
[I 140515 11:35:08 router:878] router "R6" [id=6]: ghost file set to c2691-adventerprisek9-mz.124-23.image-128.ghost
[I 140515 11:35:08 router:515] router "R1" [id=1]: has a startup-config set: "configs/R1.cfg"
[I 140515 11:35:08 router:750] router "R1" [id=1]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:515] router "R2" [id=2]: has a startup-config set: "configs/R2.cfg"
[I 140515 11:35:08 router:750] router "R2" [id=2]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:515] router "R3" [id=3]: has a startup-config set: "configs/R3.cfg"
[I 140515 11:35:08 router:750] router "R3" [id=3]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:1261] router "R3" [id=3]: adapter NM-1FE-TX inserted into slot 1
[I 140515 11:35:08 router:515] router "R4" [id=4]: has a startup-config set: "configs/R4.cfg"
[I 140515 11:35:08 router:750] router "R4" [id=4]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:1261] router "R4" [id=4]: adapter NM-1FE-TX inserted into slot 1
[I 140515 11:35:08 router:515] router "R5" [id=5]: has a startup-config set: "configs/R5.cfg"
[I 140515 11:35:08 router:750] router "R5" [id=5]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:1261] router "R5" [id=5]: adapter NM-1FE-TX inserted into slot 1
[I 140515 11:35:08 router:515] router "R6" [id=6]: has a startup-config set: "configs/R6.cfg"
[I 140515 11:35:08 router:750] router "R6" [id=6]: idle-PC set to 0x60a48df4
[I 140515 11:35:08 router:1261] router "R6" [id=6]: adapter NM-1FE-TX inserted into slot 1
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto0 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R3 [id=3] has allocated UDP port 10001 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto1 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R4 [id=4] has allocated UDP port 10002 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto2 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R5 [id=5] has allocated UDP port 10003 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto3 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R3 [id=3] has allocated UDP port 10004 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto4 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R6 [id=6] has allocated UDP port 10005 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto5 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R5 [id=5] has allocated UDP port 10006 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto6 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R4 [id=4] has allocated UDP port 10007 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:58] NIO UDP AUTO nio_udp_auto7 created with laddr=0.0.0.0, lport_start=10001, lport_end=20000
[I 140515 11:35:08 **init**:409] R6 [id=6] has allocated UDP port 10008 with host 0.0.0.0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto0 connected to 127.0.0.1:10002
[I 140515 11:35:08 router:1429] router "R3" [id=3]: NIO nio_udp_auto0 bound to port 0/1
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto1 connected to 127.0.0.1:10001
[I 140515 11:35:08 router:1429] router "R4" [id=4]: NIO nio_udp_auto1 bound to port 0/1
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto2 connected to 127.0.0.1:10004
[I 140515 11:35:08 router:1429] router "R5" [id=5]: NIO nio_udp_auto2 bound to port 1/0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto3 connected to 127.0.0.1:10003
[I 140515 11:35:08 router:1429] router "R3" [id=3]: NIO nio_udp_auto3 bound to port 1/0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto4 connected to 127.0.0.1:10006
[I 140515 11:35:08 router:1429] router "R6" [id=6]: NIO nio_udp_auto4 bound to port 0/0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto5 connected to 127.0.0.1:10005
[I 140515 11:35:08 router:1429] router "R5" [id=5]: NIO nio_udp_auto5 bound to port 0/0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto6 connected to 127.0.0.1:10008
[I 140515 11:35:08 router:1429] router "R4" [id=4]: NIO nio_udp_auto6 bound to port 1/0
[I 140515 11:35:08 nio_udp_auto:126] NIO UDP AUTO nio_udp_auto7 connected to 127.0.0.1:10007
[I 140515 11:35:08 router:1429] router "R6" [id=6]: NIO nio_udp_auto7 bound to port 1/0

Logs from gns3 running in terminal:
[lachlan@Ante-lappy ~]$ gns3
GNS3 GUI version 1.0a4
Copyright (c) 2007-2014 GNS3 Technologies Inc.
[I 140515 11:34:25 main:74] current locale is en_AU.UTF-8
libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
[I 140515 11:34:26 **init**:424] Dynamips module reset
[I 140515 11:34:26 **init**:393] IOU module reset
[I 140515 11:34:26 **init**:263] vpcs module reset
[I 140515 11:34:26 topology:136] topology has been reset
[I 140515 11:34:26 main_window:1003] creating temporary topology file: /tmp/gns3-545yp8qr
[I 140515 11:34:26 main_window:1006] creating temporary project files directory: /tmp/gns3-545yp8qr-files
[I 140515 11:34:26 **init**:173] local working directory for Dynamips module: /tmp/gns3-545yp8qr-files
[I 140515 11:34:26 **init**:138] local working directory for IOU module: /tmp/gns3-545yp8qr-files
[I 140515 11:34:26 **init**:84] local working directory for VPCS module: /tmp/gns3-545yp8qr-files
libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
[I 140515 11:35:01 servers:199] new local server connection ws://127.0.0.1:8000 registered
[I 140515 11:35:01 websocket_client:114] connected to 127.0.0.1:8000
[I 140515 11:35:01 main_window:785] use an already started local server on 127.0.0.1:8000
[I 140515 11:35:07 main_window:987] deleting temporary project files directory: /tmp/gns3-545yp8qr-files
[I 140515 11:35:07 main_window:990] deleting temporary topology file: /tmp/gns3-545yp8qr
[I 140515 11:35:07 **init**:424] Dynamips module reset
[I 140515 11:35:07 **init**:393] IOU module reset
[I 140515 11:35:07 **init**:263] vpcs module reset
[I 140515 11:35:07 topology:136] topology has been reset
[I 140515 11:35:07 main_window:962] loading project: /home/lachlan/GNS3/test/test.gns3
[I 140515 11:35:07 **init**:173] local working directory for Dynamips module: /home/lachlan/GNS3/test/test-files
[I 140515 11:35:07 **init**:138] local working directory for IOU module: /home/lachlan/GNS3/test/test-files
[I 140515 11:35:07 **init**:84] local working directory for VPCS module: /home/lachlan/GNS3/test/test-files
[I 140515 11:35:07 **init**:327] creating node <class 'gns3.modules.iou.iou_device.IOUDevice'>
[I 140515 11:35:07 **init**:152] adding server 127.0.0.1:8000 to IOU module
[I 140515 11:35:07 **init**:280] sending IOU settings to server 127.0.0.1:8000
[I 140515 11:35:07 **init**:233] opening iourc file: /home/lachlan/BTSync/Cisco/IOS/Cisco-IOU-L2-L3-Collection-v2/iourc
[I 140515 11:35:07 iou_device:44] IOU instance is being created
[I 140515 11:35:07 iou_device:515] iou device IOU1 is loading
[I 140515 11:35:07 **init**:327] creating node <class 'gns3.modules.iou.iou_device.IOUDevice'>
[I 140515 11:35:07 iou_device:44] IOU instance is being created
[I 140515 11:35:07 iou_device:515] iou device IOU2 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 **init**:187] adding server 127.0.0.1:8000 to Dynamips module
[I 140515 11:35:07 **init**:290] sending Dynamips settings to server 127.0.0.1:8000
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R1 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R2 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R3 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R4 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R5 is loading
[I 140515 11:35:07 **init**:331] creating node <class 'gns3.modules.dynamips.nodes.c2691.C2691'>
[I 140515 11:35:07 router:48] router c2691 is being created
[I 140515 11:35:07 router:839] router R6 is loading
[I 140515 11:35:07 iou_device:144] IOU instance setting up and updating console from 'None' to '4001'
[I 140515 11:35:07 iou_device:144] IOU instance setting up and updating path from '' to '/home/lachlan/BTSync/Cisco/IOS/Cisco-IOU-L2-L3-Collection-v2/i86bi-linux-l2-adventerprisek9-15.1.bin'
[I 140515 11:35:07 iou_device:144] IOU instance setting up and updating console from 'None' to '4002'
[I 140515 11:35:07 iou_device:144] IOU instance setting up and updating path from '' to '/home/lachlan/BTSync/Cisco/IOS/Cisco-IOU-L2-L3-Collection-v2/i86bi-linux-l2-adventerprisek9-15.1.bin'
[E 140515 11:35:07 iou_device:235] error while deleting IOU1: Startup-config startup-config could not be found on this server
[E 140515 11:35:07 iou_device:235] error while deleting IOU2: Startup-config startup-config could not be found on this server
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:282] router setting up and updating image from '' to '/home/lachlan/BTSync/Cisco/IOS/c2691/12.4/c2691-adventerprisek9-mz.124-23.image'
[I 140515 11:35:08 router:364] R1: updating startup_config from '' to 'configs/R1.cfg'
[I 140515 11:35:08 router:364] R1: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R1 has been updated
[I 140515 11:35:08 router:859] router R1 has been loaded
[I 140515 11:35:08 router:364] R2: updating startup_config from '' to 'configs/R2.cfg'
[I 140515 11:35:08 router:364] R2: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R2 has been updated
[I 140515 11:35:08 router:859] router R2 has been loaded
[I 140515 11:35:08 router:364] R3: updating slot1 from 'None' to 'NM-1FE-TX'
[I 140515 11:35:08 router:364] R3: updating startup_config from '' to 'configs/R3.cfg'
[I 140515 11:35:08 router:364] R3: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R3 has been updated
[I 140515 11:35:08 router:859] router R3 has been loaded
[I 140515 11:35:08 router:364] R4: updating slot1 from 'None' to 'NM-1FE-TX'
[I 140515 11:35:08 router:364] R4: updating startup_config from '' to 'configs/R4.cfg'
[I 140515 11:35:08 router:364] R4: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R4 has been updated
[I 140515 11:35:08 router:859] router R4 has been loaded
[I 140515 11:35:08 link:55] adding link from R3 FastEthernet0/1 to R4 FastEthernet0/1
[I 140515 11:35:08 router:364] R5: updating slot1 from 'None' to 'NM-1FE-TX'
[I 140515 11:35:08 router:364] R5: updating startup_config from '' to 'configs/R5.cfg'
[I 140515 11:35:08 router:364] R5: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R5 has been updated
[I 140515 11:35:08 router:859] router R5 has been loaded
[I 140515 11:35:08 link:55] adding link from R5 FastEthernet1/0 to R3 FastEthernet1/0
[I 140515 11:35:08 router:364] R6: updating slot1 from 'None' to 'NM-1FE-TX'
[I 140515 11:35:08 router:364] R6: updating startup_config from '' to 'configs/R6.cfg'
[I 140515 11:35:08 router:364] R6: updating idlepc from '' to '0x60a48df4'
[I 140515 11:35:08 router:396] router R6 has been updated
[I 140515 11:35:08 router:859] router R6 has been loaded
[I 140515 11:35:08 link:55] adding link from R6 FastEthernet0/0 to R5 FastEthernet0/0
[I 140515 11:35:08 link:55] adding link from R4 FastEthernet1/0 to R6 FastEthernet1/0

TL;DR: the iou images are not linking up for some reason.
